blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/section-nav.jsx

```diff
@@ -48,9 +47,23 @@ const isSamePath = (current, menuLink) => {
 };
 
 // $top-nav-stylistic-margin is the 13px variable in scss
-const StyledSectionMenuVariableHeight = styled.ul`
-	height: calc(100vh - ${(props) => props.navHeight}px - 13px);
-`;
+
+// TODO: This is need to be fixed properly -- instead of hardcoded
```

Comment:
Is there a specific sub task this will be addressed in? or should this be address with this first phase of removing StyledComponents?

Reply:
Will be addressed in https://arcpublishing.atlassian.net/browse/TMEDIA-888